app/Repositories/DefaultAnimeRepository.php

```diff
@@ -167,19 +167,39 @@ public function getItemsBySeason(
                 $finalFilter['$or'][] = [
                     // note: this expression only works with mongodb version 5.0.0 or higher
                     '$expr' => [
-                        '$lte' => [
+                        '$and' => [
```

Comment:
Rezero's 3rd season is still being returned in fall 2025 with this request:
`/v4/seasons/2025/winter?filter=tv&continuing&page=1`

 But doesn't this make sense given it's comprised of 16 episodes?

Reply:
This is not the case for my self-hosted version. I already have this version deployed. I'll send you a link to it to check, so we can reproduce it if this is the case.